x-pack/legacy/plugins/code/model/search.ts

```diff
@@ -8,7 +8,7 @@ import { DetailSymbolInformation } from '@elastic/lsp-extension';
 import { IRange } from 'monaco-editor';
 
 import { DiffKind } from '../common/git_diff';
-import { Repository, SourceHit } from '../model';
+import { Repository } from '../model';
```

Comment:
:information_source: There was a bug in TS that you could import something and have a different export with the same name, which has been fixed in 3.7, why this wouldn't work anymore. In this specific case we can simply remove `SourceHit` since it was not used in that file at all (but actually just declared later in this file).